README.md

```diff
@@ -2372,6 +2372,68 @@ Specify the backend URL, needed for US hosted backend infrastructure:
 yarn tr-pull-datapoints --auth=$TRANSCEND_API_KEY --file=./datapoints.csv --transcendUrl=https://api.us.transcend.io
 ```
 
+### tr-pull-pull-unstructured-discovery-files
+
+This command allows for pulling Unstructured Discovery into a CSV.
+
+#### Authentication
+
+In order to use this cli, you will first need to generate an API key on the Transcend Admin Dashboard (https://app.transcend.io/infrastructure/api-keys).
+
+The API key must have the following scopes:
+
+- "View Data Inventory"
+
+#### Arguments
+
+| Argument                 | Description                                                                                          | Type          | Default                            | Required |
+| ------------------------ | ---------------------------------------------------------------------------------------------------- | ------------- | ---------------------------------- | -------- |
+| auth                     | The Transcend API key with the scopes necessary for the command.                                     | string        | N/A                                | true     |
+| file                     | The file to save datapoints to                                                                       | string - path | ./unstructured-discovery-files.csv | false    |
+| transcendUrl             | URL of the Transcend backend. Use https://api.us.transcend.io for US hosting.                        | string - URL  | https://api.transcend.io           | false    |
+| dataSiloIds              | Comma-separated list of data silo IDs to filter by                                                   | string        | N/A                                | false    |
+| subCategories            | Comma-separated list of data categories to filter by                                                 | string        | N/A                                | false    |
+| status                   | Comma-separated list of classification statuses to filter by (CLASSIFIED, VALIDATED, REJECTED, etc.) | string        | (excludes REJECTED)                | false    |
```

Comment:
@michaelfarrell76 in the GraphQL interface, if no `status` is provided, it will exclude REJECTED recommendations by default. I wasn't sure how best to indicate that. How's this?

Reply:
looks great!